### PR TITLE
GUBMAN 2 & 2.5 REVISITED: REMASTERED HD

### PR DESCRIPTION
**Pull request by Draganfrukts**
## About The Pull Request

gubmanfunny

This PR revisits aspects of Gubman 2 and Gubman 2.5.

## Why It's Good For The Game

Sprite updates, less useless stuff in code.

## Changelog
:cl:
imageadd: Sniper rifle sprite resolution changed to 48x32 from 40x32
tweak: Hardlight scopes were deemed too expensive to produce. Sniper rifles are once again fit with common optics. Some may still have electronic assistance...
tweak: All old sniper rifle models were phased out of the market. New designs will be used, regardless of manufacturer.
tweak: Super & Hyper kinetic accelerators have custom shapes instead of being recolors.
imageadd: MCRS-5B and ARG-63 have been re-sprited.
imageadd: AUS-107, SA-107 and Caracal AMR have been re-sprited.
/:cl:

### CODE REVIEW

**Comment by Draganfrukts** [comment]
Conflict fixed. Das Rite!

**Comment by Draganfrukts** [comment]
fuck

**Comment by Draganfrukts** [comment]
**AAAAAAAAAAGH STUPID FUCKING MONKEY!!!!!!!**

**Comment by Draganfrukts** [comment]
i'M THE GREATEST PROGRAMMER THAT HAS EVER LIVED.
imagen

File: code/modules/projectiles/guns/energy/kinetic_accelerator.dm [hunk 1]
```diff
@@ -569,7 +569,7 @@
 	KA.name = initial(KA.name)
 	..()
 
-/obj/item/borg/upgrade/modkit/chassis_mod/orange
+/obj/item/borg/upgrade/modkit/chassis_mod/orange                   /////ICON OVERRIDE IN modular_skyrat/modules/aesthetics/guns/icons/energy.dmi
```
Comment (Gandalf2k15) [inline code/modules/projectiles/guns/energy/kinetic_accelerator.dm]:
```suggestion
/obj/item/borg/upgrade/modkit/chassis_mod/orange /////ICON OVERRIDE IN modular_skyrat/modules/aesthetics/guns/icons/energy.dmi
```

File: code/modules/projectiles/guns/energy/kinetic_accelerator.dm [hunk 2]
```diff
@@ -550,7 +550,7 @@
 
 //Cosmetic
 
-/obj/item/borg/upgrade/modkit/chassis_mod
+/obj/item/borg/upgrade/modkit/chassis_mod                   /////ICON OVERRIDE IN modular_skyrat/modules/aesthetics/guns/icons/energy.dmi
```
Comment (Gandalf2k15) [inline code/modules/projectiles/guns/energy/kinetic_accelerator.dm]:
```suggestion
/obj/item/borg/upgrade/modkit/chassis_mod /////ICON OVERRIDE IN modular_skyrat/modules/aesthetics/guns/icons/energy.dmi
```

**Review by Gandalf2k15 — CHANGES_REQUESTED** [review]

**Comment by Draganfrukts** [comment]
See Gandalf you broke it

**Comment by Gandalf2k15** [comment]
FIX IT

**Comment by Draganfrukts** [comment]
YOU COULD'VE MERGED THIS BEFORE FUCKING BREAKING IT IT'S FULL AUTO'S FAULT, YOU FIX IT

**Review by Gandalf2k15 — APPROVED** [review]

**Pull request merged**